fa_eval/README.md

```diff
@@ -1,22 +1,89 @@
-#### TimeFrame+Alignment evaluation code
+# TimeFrame+Alignment evaluation code
+## Theme: Forced Alignment 
+This is the alignment of the text to time positions for transcription/closed-captioning. 
+This tool processes human-annotated(`.tsv`) and machine-generated(`.mmif`) and outputs the evaluation results.
 
-This tool processes human-annotated(.srt) and machine-generated(.mmif) and outputs the evaluation results.
+To run the evaluation you will need matching prediction `.mmif` and gold `.tsv` files
 
-#### Parameters:
--m or --machine_dir: Directory containing machine annotated .mmif files.
+It is done by (tba).   
+It is evaluated by (tba). 
 
--o or --gold_dir: Directory containing human annotated .srt files.
+## Parameters
```

Comment:
I really don't think having this CLI arguments separately documented in the readme is a good idea. These should just stay in the code itself, and we can simply say here that users can refer to `python <script> --help` to get the documentation.

Reply:
Got it. The intent here is to make it more explicit which files are required as each project is different. I have updated the `parser.add_argument` sections to reflect the expected filetypes. (They are hardcoded typewise). Does that sound good?